src/ide/editor/language.ts

```diff
@@ -191,7 +198,7 @@ export class Language {
 				let set = new Set<number>();
 				while (true) {
 					let begin = rule.charCodeAt(pos);
-					if (begin === " ") break;
+					if (begin === 0x20) break; // Space
```

Comment:
`charCodeAt` returns a number. This might be a behavioural change since the loop was never exited here. Debugging indicated that spaces might've previously been caught by `if (rule[pos] !== "+") break;` below.